.github/workflows/integrations.yml

```diff
@@ -98,6 +98,18 @@ jobs:
       - name: Run integration build
         run: |
           ./tests/ci/integration/run_socat_integration.sh
+  nmap:
+    if: github.repository_owner == 'aws'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install OS Dependencies
+        run: |
+          sudo apt-get update -o Acquire::Languages=none -o Acquire::Translation=none
+          sudo apt-get -y --no-install-recommends install cmake gcc ninja-build golang make cairo libcario2 libcairo2-dev gir1.2-gtk-3.0 python3-gi gobject-introspection libgirepository1.0-dev
```

Comment:
Hmmm interesting I've been testing this on a ubuntu system as well. Since the packages aren't available in the github runner I can disable zenmap but I wonder if that leaves a compatibility gap? I guess it's just the UI though. (could try pip installing them as a part of the integration script since they aren't available through the system as well).

Reply:
i don't think we need to worry about the GUI until/if we learn of customer need.